pkg/schedule/operator/operator_controller.go

```diff
@@ -575,11 +575,27 @@ func (oc *Controller) removeOperatorLocked(op *Operator) bool {
 		oc.updateCounts(oc.operators)
 		operatorCounter.WithLabelValues(op.Desc(), "remove").Inc()
 		oc.ack(op)
+		if op.Kind()&OpMerge != 0 {
+			oc.removeRelatedOperator(op)
+		}
 		return true
 	}
 	return false
 }
 
+func (oc *Controller) removeRelatedOperator(op *Operator) {
```

Comment:
```suggestion
func (oc *Controller) removeRelatedMergeOperator(op *Operator) {
```

Reply:
Okay, This name is because for the two operators of hot-region and other cases, we may need to cancel together as well. So I hope to unify name for removing related func.
But for now we just need to be specific.

Reply:
done

Reply:
done